struct_env/struct_env.py

```diff
@@ -77,18 +135,27 @@ def step(self, action: dict):
         self.observations = {}
         for i in range(self.ncomp):
             self.observations[self.agent_list[i]] = np.concatenate(
-                (belief_prime[i], [self.time_step / 30]))
+                (self.beliefs[i], [self.time_step / 30]))
+
+        if self.env_type is "correlated":
+            for i in range(self.ncomp):
+                self.observations[self.agent_list[i]] = np.concatenate(
+                    (self.observations[self.agent_list[i]], self.alphas, [self.time_step / 30]))
```

Comment:
I have also added the normalized timestep here.

Reply:
[resolved here](https://github.com/moratodpg/ray_environments/pull/7/commits/443a433a717db4448eb9d2543cb0b0e6cc089df9#diff-df106087ea8381e5153ab17bdf66cdefe4b25af00235ab7cb504fdd43ad5e37eR144)